tests/test_gui.py

```diff
@@ -0,0 +1,904 @@
+    app_dialog.root.listitems.mouseDoubleClick()
+    app_dialog.root.captions["Azure's Note on Toolkit Panel UI Automation"].waitExist(
+        timeout=30
+    )
+    assert app_dialog.root.captions[
+        "New note created by automation"
+    ].exists(), "New Note is missing"
+    assert app_dialog.root.captions[
+        "Note by Azure Pipelines*Written on*Addressed to: Azure Pipelines*Associated With:*Project Toolkit Panel UI Automation*"
+    ].exists(), "Not the Notes details"
+
+    # Create a reply
+    app_dialog.root.captions["Reply to this Note"].mouseClick()
+    app_dialog.root.dialogs["Reply"].waitExist(timeout=30)
+
+    # Validate that all buttons are available
+    assert (
+        app_dialog.root.dialogs["Reply"].buttons["Cancel"].exists()
+    ), "Cancel buttons is not showing up"
+    assert (
+        app_dialog.root.dialogs["Reply"].buttons["Attach Files"].exists()
+    ), "Attach Screenshot buttons is not showing up"
+    assert (
+        app_dialog.root.dialogs["Reply"].buttons["Take Screenshot"].exists()
+    ), "Take Screenshot buttons is not showing up"
+    assert (
+        app_dialog.root.dialogs["Reply"].buttons["Create Note"].exists()
+    ), "Create Note buttons is not showing up"
+
+    # Validate that the File browser is showing up after clicking on the Files to attach button then close it
+    app_dialog.root.dialogs["Reply"].buttons["Attach Files"].mouseClick()
+    app_dialog.root.dialogs["Select files to attach."].waitExist(timeout=30)
+
+    # Get image path to be published
+    image_path = os.path.normpath(
+        os.path.expandvars("${TK_TEST_FIXTURES}/files/images/sven.png")
+    )
+
+    # Type in image path
+    app_dialog.root.dialogs["Select files to attach."].textfields[
+        "File name:"
+    ].mouseClick()
+    app_dialog.root.dialogs["Select files to attach."].textfields["File name:"].pasteIn(
+        image_path
+    )
+    app_dialog.root.dialogs["Select files to attach."].textfields[
+        "File name:"
+    ].waitIdle(timeout=30)
+    app_dialog.root.dialogs["Select files to attach."].textfields["File name:"].typeIn(
+        "{ENTER}"
+    )
+
+    # Validate that all buttons are available
+    assert (
+        app_dialog.root.dialogs["Reply"].buttons["Cancel"].exists()
+    ), "Cancel button is not showing up"
+    assert (
+        app_dialog.root.dialogs["Reply"].buttons["add_button"].exists()
+    ), "Add attachments button is not showing up"
+    assert (
+        app_dialog.root.dialogs["Reply"].buttons["remove_button"].exists()
+    ), "Remove attachments button is not showing up"
+    assert (
+        app_dialog.root.dialogs["Reply"].buttons["Create Note"].exists()
+    ), "Create Note button is not showing up"
+    app_dialog.root.dialogs["Reply"].buttons["Create Note"].mouseClick()
+
+    # Take a screenshot
+    app_dialog.root.dialogs["Reply"].buttons["Take Screenshot"].mouseClick()
+    MyOGL = first(app_dialog.root)
```

Comment:
nitpick, this doesn't follow the variable naming convention.  Also what does my OGL stand for?

Reply:
oups... this is a copy/paste from a MA.UI example I found to how to do a screen selection and I forgot to update it.